packages/components/psammead-podcast-promo/src/components/card-image.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { number, string } from 'prop-types';
+
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+
+const Wrapper = styled.figure`
+  width: 88px;
+  display: inline-block;
+  vertical-align: top;
+  margin: 8px 0 0 8px;
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    width: 109px;
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: 178px;
+    margin: 0;
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    width: 100%;
+    margin: 0;
+  }
+  img {
```

Comment:
should we avoid nested selectors where we can?

Reply:
👍 Updated `img` to use its own styled component.  Can you explain (or link something that explains) why this is better?

Reply:
Nesting increases specificity and means we can't restructure HTML without worrying about messing up styles. Also it's better for performance if not nested since browsers evaluate selectors from right to left. In this example, all img elements would be selected first and then the figure element. Element selectors are not great either especially if we decided to use a picture element instead of an img element

Reply:
Oh wow - didn't know about the right to left thing - so browsers would select all images on the page, and then filter them for ones that are children of the left selector

That's crazy - thanks!